controller/src/beerocks/master/tasks/dynamic_channel_selection_task.cpp

```diff
@@ -0,0 +1,131 @@
+/* SPDX-License-Identifier: BSD-2-Clause-Patent
+ *
+ * Copyright (c) 2016-2019 Intel Corporation
```

Comment:
New file - 2020

Reply:
If the file was first created (and distributed to customers, even if in binary form) in 2016, then the starting copyright year really should be 2016.

Reply:
it was created in in 2019 in intel repos so i changed to 2019-2020